tests/utils/content.js

```diff
@@ -0,0 +1,123 @@
+const {
+  getAgeGroupIdForActivity,
+  getAgeGroupIdFromActivityGroup,
+  isObject,
+} = require("../../utils/content");
+
+describe("Content utils", () => {
+  let activities;
+  let activityGroups;
+  let ageGroup;
+
+  beforeAll(async () => {
+    /*
+      Create content hierarchy for testing:
+      - ageGroup
+        - activityGroup3
+          - activity-3
+          - activityGroup2
+            - activity2
+            - activityGroup1
+              - activity1
+    */
+    const activity1 = await strapi.query("activity").create({
```

Comment:
Test data should also be able to differentiate, now activity 1 2 and 3 all have same content. Maybe change data to include also number in variable name, so we can differentiate those.

Reply:
The content has no meaning in these tests. The point of interest is in the relations, which are verified by the id's.